enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/entity/OnboardingProcessEntity.java

```diff
@@ -60,8 +60,8 @@ public class OnboardingProcessEntity implements Serializable {
     public static final String ERROR_USER_LOOKUP = "userLookupFailed";
 
     @Id
-    @GeneratedValue(generator = "uuid")
-    @GenericGenerator(name = "uuid", strategy = "uuid2")
+    @GeneratedValue
+    @UuidGenerator(style = UuidGenerator.Style.TIME)
```

Comment:
Are you sure, that `Style.TIME` is equivalent of `uuid2`? I hesitate when I see the deprecated `UUIDGenerationStrategy` where `2 = DCE based using POSIX UIDs`. Version 2 is time-based but should also include MAC https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_2_(date-time_and_MAC_address,_DCE_security_version)

Reply:
You are probably correct, it corresponds to UUID v1. I am trying to find a solution now in the documentation

Reply:
Isn't it version 4?

Reply:
See this the default strategy is AUTO -> RANDOM which triggers randomUUID() corresponding to v4 UUID
https://docs.jboss.org/hibernate/orm/6.2/javadocs/org/hibernate/annotations/UuidGenerator.Style.html
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/UUID.html?is-external=true#randomUUID()